examples/debugger/src/main.rs

```diff
@@ -8,19 +9,52 @@ use sunscreen::{
 };
 
 fn main() {
+    #[debug_program]
```

Comment:
I think we should just call this `debug` and users can do `sunscreen::debug` to be clear about what this is.